ui/src/routes/_layout/organizations/$orgId/products/$productId/repositories/$repoId/_repo-layout/runs/index.tsx

```diff
@@ -47,11 +47,11 @@ import {
 } from '@/components/ui/tooltip';
 import { toast } from '@/lib/toast';
 import { paginationSearchParameterSchema } from '@/schemas';
-import { RepositoryRunsTable } from '../-components/repository-runs-table';
+import { RepositoryRunsTable } from '../../-components/repository-runs-table';
```

Comment:
Should this now actually be an absolute link `@/...` as it's not anymore "close" to the page where it's used?

Reply:
> Should this now actually be an absolute import `@/...` as it's not anymore "close" to the page where it's used?

I think it's still relatively close, as the absolute import would be `@/routes/_layout/organizations/$orgId/products/$productId/repositories/$repoId/-components/repository-runs-table`. What do you think?

Reply:
Well, yeah, probably better keep it as-is.